scripts/linters/pylint_extensions_test.py

```diff
@@ -696,29 +692,28 @@ def test_malformed_raises_section(self):
             self.checker_test_object.checker.visit_functiondef(
                 node_malformed_raises_section)
 
-    def test_malformed_args_argument(self):
-        node_malformed_args_argument = astroid.extract_node(
-            u"""def func(*args): #@
+    def test_presence_of_typeinfo_in_args_section_considered_as_incorrect(self):
+        node_malformed_args_section = astroid.extract_node(
+            u"""def func(arg: str) -> None: #@
```

Comment:
Right now what error is raised if `*args` and `**kwargs` are present and the docstring is

```
Args:
  args: Set of arguments.
  kwargs: Set of kwargs
```

Reply:
Thanks for pointing out this case, Previously linters are throwing `W9020: Line starting with "args:" requires 8 space indentation relative to args line indentation` but after adding new changes, linters are throwing `core/controllers/acl_decorators.py:818:4: W9015: "**kwargs, *args" missing in parameter documentation 
core/controllers/acl_decorators.py:818:4: W9017: "args, kwargs" differing in parameter documentation 
`